src/site/content/en/blog/declarative-link-capturing/index.md

```diff
@@ -152,6 +154,58 @@ possible for the site to automatically trigger it. Second, the capability to foc
 window on its own domain and fire an event containing the clicked URL. This is intended to allow the
 site to navigate an existing window to a new page, overriding the default HTML navigation flow.
 
+## Migrate to Launch Handler API {: #migration }
+
+The Declarative Link Capturing API
+[origin trial](https://developer.chrome.com/origintrials/#/view_trial/4285175045443026945) is set to
+[expire on March&nbsp;30, 2022](https://groups.google.com/a/chromium.org/g/blink-dev/c/2c4bul4V3GQ/m/Anluh1txBQAJ)
+for Chromium&nbsp;97 and below. It will be replaced by a set of
+[new features and APIs](https://docs.google.com/document/d/1w9qHqVJmZfO07kbiRMd9lDQMW15DeK5o-p-rZyL7twk/edit)
+in Chromium&nbsp;98 and above, which includes user-enabled link capturing and
+[Launch Handler API](https://github.com/WICG/sw-launch/blob/main/launch_handler.md).
+
+### Link Capturing
+
+In Chromium&nbsp;98, automatic link capturing is now a user opt-in behavior rather than granted at
+install time to a web app. To enable link capturing, a user needs to launch an installed app from
+the browser using **Open with** and choose **Remember my choice**.
+
+{% Img src="image/ttTommHYbJXsEL29zNB1wXBvH4z1/rbFk5LtzHMlN3zRVpekf.png", alt="Example of an installed app's 'Open with' setting with the 'Remember my choice' option enabled.", width="385", height="277" %}
+
+Alternatively, users can switch link capturing on or off for a specific web app in the app
+management settings page.
+
+{% Img src="image/ttTommHYbJXsEL29zNB1wXBvH4z1/WJ2KPqFUjqJABNYQUEN9.png", alt="Example of an installed app's settings page.", width="800", height="449" %}
+
+Link capturing is a Chrome&nbsp;OS-only feature for now; support for Windows, macOS, and Linux is
+still in progress.
+
+### Launch Handler API
+
+The control of an incoming navigation is migrated to Launch Handler API, which allows web apps to
+decide how a web app launches at various situations, such as link capturing, share target or file
+handling, etc. To migrate from the Declarative Link Capturing API to the Launch Handler API:
+
+1.  Register your site for the
+    [Launch Handler origin trial](https://developer.chrome.com/origintrials/#/view_trial/2978005253598740481)
+    and place the origin trial key into your web app.
+1.  Add a `"launch_handler"` entry to your site's manifest.
+
+    - If you are using `"capture_links": "new-client"`, then
+      add:`"launch_handler": { "route_to": "new-client" }`
+    - If you are using `"capture_links": "existing-client-navigate"`, then add:
+      `"launch_handler": { "route_to": "existing-client" }`
+    - If you wanted to use `"capture_links": "existing-client-event"` (which was never implemented
+      in the Declarative Link Capturing origin trial), then add:
+      `json { "launch_handler": { "route_to": "existing-client", "navigate_existing_client": "never" } } `
```

Comment:
Why is this one phrased differently?

>Which was never implemented...

Why is this here? That we didn't implement it in the link capture trial isn't really relevant to the launch handler trial. Drawing attention to the fact that we didn't implement it could give the impression that we still regard it as a second class citizen.

Reply:
This one will need to change to match the other two.

Reply:
I think the rhythm was broken since this was never usable.

Reply:
I don't follow.

Reply:
Aligned the rhythm. This should work now.

Reply:
The rhythm is right. (The term of art among writers is parallelism.) 

For this to stay, it should be in positive form. 'New in this version' as opposed to 'not previously implemented'. We care that it's here now, not that it wasn't implemented before.

Reply:
This is a statement about the past, though. The explainer said it was there and the article documented it, but the implementation never supported it. The goal of this paragraph is to steer people away from the old stuff (declarative link capturing) and toward the new stuff (launch handler). So we want the old stuff to look worse than the new stuff.

Reply:
I know it's about the past. I'm saying it shouldn't be.

Reply:
Then please change it so that the sense is preserved.